CHANGELOG.md

```diff
@@ -22,10 +22,9 @@
 
 #### Enhancements
 
-* Adds `discouraged_optional_boolean` opt-in rule to discourage
-  the use of optional booleans.  
-  [Ornithologist Coder](https://github.com/ornithocoder)
-  [#2011](https://github.com/realm/SwiftLint/issues/2011)
+* Adds `function_body_whitespace_comment` opt-in rule to prohibit empty and comment lines in function bodies.  
```

Comment:
Please note the length of these lines [should not exceed 80 columns](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes).